Includes/menu.cpp

```diff
@@ -130,6 +131,31 @@ void MenuXbe::execute(Menu *menu) {
   }
 }
 
+/******************************************************************************************
+                                   MenuThemes
+******************************************************************************************/
+
+MenuThemes::MenuThemes(MenuNode *parent, std::string const& label, std::string const& path) :
+  MenuNode(parent, label), path(path) {
+  findThemes(path, this);
+
+}
+
+MenuThemes::~MenuThemes() {
+
+}
+
+void MenuThemes::execute(Menu *menu) {
+  if (menu->getCurrentMenu() != this) {
+    menu->setCurrentMenu(this);
+  }
+  else {
+    if (childNodes.size() > selected) {
+      this->childNodes.at(selected)->execute(menu);
+    }
+  }
+}
```

Comment:
This doesn't do anything different from `MenuNode`, does it?
In that case, this can be removed.

I'd wager you want `ThemeItem` to perform the actual magic in the end anyways?